lib/date_time_parser.ex

```diff
@@ -132,23 +130,37 @@ defmodule DateTimeParser do
   If a time cannot be determined, then it will not be assumed by default. If you supply `true`, then
   `~T[00:00:00]` will be assumed. You can also supply your own time, and the found tokens will be
   merged with it.
+
+  * `:parsers` Default `[:epoch, :serial, :tokenizer]`.
```

Comment:
Is it possible to use string interpolation and [the module attribute in `Parser`](https://github.com/taxjar/date_time_parser/pull/22/files#diff-3d3bba1d594bbc36c472139cd6484460R7) to get the list of parsers dynamically?

Reply:
good idea! Do you know how to do that in exdoc? I remember seeing that in a project but can't remember where.

Reply:
I tried doing some quick googling and didn't see anything solid. [This issue](https://github.com/elixir-lang/ex_doc/issues/440) makes it seem like string interpolation may work.

Reply:
Looks like it's standard string interpolation `"my string #{foo}"`:

```elixir
@doc """
yada yada
* `:parsers` Default `#{inspect DateTimeParser.Parser.parsers()}`.
"""
```

![image](https://user-images.githubusercontent.com/643967/71562529-a65b3680-2a4f-11ea-843d-85e046348084.png)